src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingService.kt

```diff
@@ -301,41 +301,40 @@ class SentencingService(
     eventId: Long,
     request: UpdateCourtAppearanceRequest,
   ) {
-    findPrisoner(offenderNo).findLatestBooking().let { booking ->
```

Comment:
Formatting has made this confusing - essentially just removing the findLatestBooking as this can be determined from the court case